contracts/interfaces/IStakingPool.sol

```diff
@@ -162,4 +163,5 @@ interface IStakingPool {
 
   event PoolFeeChanged(address indexed manager, uint newFee);
 
+  event PoolDescriptionSet(uint poolId, string ipfsDescriptionHash);
```

Comment:
@mixplore  assuming it was clarified you are polling each staking pool in turn for these description events and that's fine for the frontend